.github/workflows/deploy-quay.yml

```diff
@@ -49,17 +49,17 @@ jobs:
         env:
           GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
 
-      - name: Load Docker image to local registry
-        run: |
-          docker load -i ${ARTIFACT_PATH}
-          docker tag ${PAYLOAD_NAME}:${PAYLOAD_VERSION} quay.io/${PAYLOAD_REPOSITORY}:${PAYLOAD_VERSION}
-
       - name: Log in to Quay Container Registry
         uses: docker/login-action@v3
         with:
           registry: quay.io
           username:  ${{ secrets.QUAY_USERNAME }}
           password: ${{ secrets.QUAY_PASSWORD }}
 
+      - name: Install Skopeo
```

Comment:
Just found out that Skopeo is already installed in the Github Actions by default: https://github.com/actions/runner-images/blob/main/images/ubuntu/Ubuntu2404-Readme.md